src/browser/AxePage.ts

```diff
@@ -17,14 +17,36 @@ export function analyze(
   page: Page,
   disabledRules: string[] = [],
   runOptions: RunOptions = {},
+  config?: Spec,
 ): Promise<AxeResults> {
-  return page.evaluate(runAxe, getRunOptions(runOptions, disabledRules));
+  return page.evaluate(runAxe, {
+    options: getRunOptions(runOptions, disabledRules),
+    config,
+  });
 }
 
-function runAxe(options: RunOptions): Promise<AxeResults> {
+function runAxe({
+  options,
+  config,
+}: {
+  options: RunOptions;
+  config?: Spec;
+}): Promise<AxeResults> {
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore This function executes in a browser context.
-  return window.axeQueue.add(() => window.axe.run(document, options));
+  return window.axeQueue.add(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore This function executes in a browser context.
+    window.axe.reset();
```

Comment:
Need to `reset` so that one story doesn't affect the others. This is captured in the integration test.

Reply:
For maintenance purposes, I wonder if a comment like this is good to have in the code. But a test captures this so may not be needed

Reply:
I think a comment makes sense. Will add.